packages/web/index.tsx

```diff
@@ -0,0 +1,10 @@
+import * as JigsawUI from "../react-native-jigsaw/src/index";
```

Comment:
with workspaces you shouldn't need to do this, right? you should be able to do `import * as JigsawUI from '@draftbit/ui'` right?

Reply:
I could do `"@draftbit/ui"` but then I would rely on whatever is under the "main" package.json for UI, so I explicitly make that dependency

Happy to annotate that or change it to rely on "main".